lightstep/resource_metric_dashboard.go

```diff
@@ -191,41 +182,80 @@ func getChartSchema(chartSchemaType ChartSchemaType) map[string]*schema.Schema {
 			Default:      0,
 			Optional:     true,
 		},
-		"type": {
-			Type:         schema.TypeString,
-			Required:     true,
-			ValidateFunc: validation.StringInSlice([]string{"timeseries"}, true),
-		},
 		"id": {
 			Type:     schema.TypeString,
 			Computed: true,
 		},
-		"y_axis": {
-			Type:       schema.TypeList,
-			MaxItems:   1,
-			Deprecated: "The y_axis field is no longer used",
+	}
+}
+
+func getChartSchema(chartSchemaType ChartSchemaType) map[string]*schema.Schema {
```

Comment:
`getChartSchema` is unchanged other than moved code and splitting out the common schema to a call to `getPanelSchema`.